docs/data-builder-dummy-data.md

```diff
@@ -0,0 +1,39 @@
+# Supplying a dummy dataset
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+Because OpenSAFELY doesn't allow direct access to individual patient
+records, researchers must use a **dummy dataset** for developing their analytic
+code on their own computer.
+
+A dummy dataset needs to be similar enough to a real dataset to allow
+the analysis scripts to run successfully, but it must be randomly (or
+psuedo-randomly) generated and not contain data relating to real
+patients.
+
+In future, Data Builder will be able to generate suitable dummy data
```

Comment:
question: are we guaranteeing this?

Reply:
suggestion: Perhaps consider splitting this sentence into two (until "but for now") and maybe putting the first part in a `!!! note`.